Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM python:3
+ADD src/quarto.py /
+RUN pip install pystrich
+RUN pip install pep8
```

Comment:
Have a look on `pipenv` instead of using `pip` directly. Passing it a `PIPENV_VENV_IN_PROJECT` environment variable, you can embed the vendor dependencies in your project root, instead of in an obscure part of your system.

![image](https://user-images.githubusercontent.com/688373/40927406-ef9beb72-681e-11e8-9a73-12c23af4b6cd.png)

Reply:
I didn't modify that for now, considering the code works without and it's a small project with short delay
But I keep it in mind for next